templates/article_page.hbs

```diff
@@ -210,16 +210,38 @@
                             {{#link 'user_profile' id=author.id}}
                               {{author.name}}
                             {{/link}}
+                            {{#each author.badges}}
+                              {{#is category_slug "titles"}}
+                                <span class="community-badge community-badge-{{category_slug}}" title="{{description}}">
```

Comment:
nitty nit: since we are checking for the `category_slug` value before, we can make the classname explicit so it's searchable

```suggestion
                                <span class="community-badge community-badge-titles" title="{{description}}">
```